### PR TITLE
Adi laqma other mss

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Further mss of Addi Laqma Maryam.

### CODE REVIEW

File: AlaqmaMaryamZadibaKokweh/MALZ017.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MALZ017" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Lectonary</title>
```
Comment (eu-genia) [inline AlaqmaMaryamZadibaKokweh/MALZ017.xml:8]: io

File: AlaqmaMaryamZadibaKokweh/MALZ017.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1012AddiLaqmaMaryam"/>
+                        <idno>MALZ-017</idno>
+                        </msIdentifier>
+                    <msContents>
+                       <summary/>
+                     
+                        <msItem xml:id="ms_i1">                        
+                            <title type="incomplete" ref="LIT1917Mashaf">Lectonary</title>
```
Comment (eu-genia) [inline AlaqmaMaryamZadibaKokweh/MALZ017.xml:34]: io

**Review by eu-genia — APPROVED** [review]

File: AlaqmaMaryamZadibaKokweh/MALZ017.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply AlaqmaMaryamZadibaKokweh/MALZ017.xml]: ok

File: AlaqmaMaryamZadibaKokweh/MALZ017.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply AlaqmaMaryamZadibaKokweh/MALZ017.xml]: ok

**Pull request merged**